tailwind.config.ts

```diff
@@ -43,18 +44,18 @@ export default {
 			'chocolate-cosmos': '#520c13',
 			'upsdell-red': '#a71d2a',
 			'alice-blue': '#ecf3fb',
-			'american-orange': '#ff8a00',
 			'crayola-yellow': '#ffe57f',
 			'warning-orange': '#FFF3E6',
 			cornsilk: '#fff7d8',
-			cobalt: '#004abe',
 			'resolution-blue': '#012f80',
 			zumthor: '#e8f1ff',
 			onahau: '#d1e3ff',
 			anakiwa: '#b0cdff',
 			beer: '#f7931a',
 			fulvous: '#de7900',
-			water: '#d1e3ff'
+			water: '#d1e3ff',
+			...colorsBase,
```

Comment:
@AntonioVentilii-DFINITY  We should not add the colorsBase here: You don't want people to use colors directly, they should always be referenced via a theme variable (eg. foreground-brand-primary) because otherwise theming will be impossible. 
Example: If someone uses a certain blue directly, instead of assigning the theme-mapped-variable, this component then always will be blue, no matter which theme is applied.

Reply:
Agree, done